### PR TITLE
Visitor can see ads

### PR DESCRIPTION
**Pull request by erikbjoern**
Screenshot from 2020-05-23 15-13-01
Screenshot from 2020-05-23 15-14-00
Screenshot from 2020-05-23 15-13-26

## User story
[PT link](https://www.pivotaltracker.com/story/show/172888064)
```
As a Visitor,
In order to get great offers,
I would like to see ads
```
# Feature
* Visitor can see ads
* - On main page
* - On single article page

### CODE REVIEW

**Review by kermit-klein — APPROVED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from "react";
-import { 
-  BrowserRouter as Router,
+import {
```
Comment (oliverochman) [inline src/App.jsx]: In the future try to avoid changes that are out of scope

File: src/components/Ads2.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Grid } from "semantic-ui-react";
+import lagavulinAd from "../images/lagavulinAd.jpg";
+
+const Ads2 = () => {
```
Comment (oliverochman) [inline src/components/Ads2.jsx]: it is unnecessary to have seperate components

reuse the same component

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/components/Ads2.jsx (hunk 2 above)
Comment (kermit-klein) [reply src/components/Ads2.jsx]: We have one component now serving two different ads

File: src/App.jsx (hunk 1 above)
Comment (kermit-klein) [reply src/App.jsx]: hmm we deleted something we don't use in this file

File: src/components/Ad.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Grid } from "semantic-ui-react";
+import mercedesImg from "../images/mercedesAd.jpg";
+import lagavulinImg from "../images/lagavulinAd.jpg";
+
+const Ad = ({ type }) => {
+  const adList = [
+    {
+      id: "ad-1",
+      href: "https://www.mercedes-benz.com/en/",
```
Comment (oliverochman) [inline src/components/Ad.jsx]: pass in this as props instead

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/components/Ad.jsx (hunk 3 above)
Comment (ViaMarcus) [reply src/components/Ad.jsx]: Would reading an external json work fine? Would make editing it easier at least for owner. We are selecting from the list using a prop right now.

**Review by jysmys — APPROVED** [review]

File: src/App.jsx (hunk 1 above)
Comment (designerofthing) [reply src/App.jsx]: When should something like this be deleted or modified?  I thought if you find a mistake you can fix it there and then, do you need to go back and change branches to do this?

**Review by designerofthing — APPROVED** [review]
Looks good

**Review by PaulineBA — APPROVED** [review]

**Pull request merged**